src/utilities/util_dump.c

```diff
@@ -37,22 +38,25 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
     WT_DECL_RET;
     WT_SESSION_IMPL *session_impl;
     int ch, i;
-    char *checkpoint, *ofile, *p, *simpleuri, *uri;
+    char *checkpoint, *ofile, *p, *simpleuri, *timestamp, *uri;
     bool hex, json, reverse;
 
     session_impl = (WT_SESSION_IMPL *)session;
 
     cursor = NULL;
-    checkpoint = ofile = simpleuri = uri = NULL;
+    checkpoint = ofile = simpleuri = uri = timestamp = NULL;
     hex = json = reverse = false;
-    while ((ch = __wt_getopt(progname, argc, argv, "c:f:jrx")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "c:f:t:jrx")) != EOF)
```

Comment:
We need to update the WiredTiger docs for this change, I think.

Reply:
done!